expyfun/stimuli/_hrtf.py

```diff
@@ -40,29 +39,82 @@ def _get_hrtf(angle, source, fs):
     0-degree BRIRs (mean of that across channels) is equal to 1. This will
     ensure that the RMS of a white signal filtered with this signal is
     unchanged.
+
+    If interp is True, the function takes the two nearest known HRTFs and uses
+    them to calculate an HRTF between them for any given azimuthal angle
+    between 0 and 90 degrees.
+
+    Interpolation relies on averaging log magnitude and phase in the frequency
+    domain. [1]
```

Comment:
@maddycapp27 I think you still missed this change.  The underscore after `[1]` is necessary for sphinx to create the hyperlink.